Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -169,88 +181,129 @@ def get_thrust_level_and_score(reputations):
-            'ContentsFormat': formats['json'],
-            'Contents': reputations,
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable': tableToMarkdown('McAfee TIE Hash Reputations For %s:' % (hash_value,), table),
-            'EntryContext': ec
-        })
-    return hash_list
+                reputations = raw_result.values()
+                
+                tl_score = get_trust_level_and_score(reputations)
+                trust_level = tl_score['trust_level']
+                vendor = tl_score['vendor']
+                score = tl_score['score']
+
+                dbot_score_instance = Common.DBotScore(indicator=file_hash,
+                                                       indicator_type=DBotScoreType.FILE,
+                                                       reliability=dbot_reliability,
+                                                       integration_name=VENDOR_NAME,
+                                                       score=score,
+                                                       )
+                file_instance = Common.File(dbot_score=dbot_score_instance)
+
+                dbot_score = {'Indicator': file_hash,
+                              'Type': 'hash',
+                              'Vendor': vendor,
+                              'Score': score,
+                              'Reliability': demisto.params().get('integrationReliability', 'C - Fairly reliable'),
+                              }
+                file['TrustLevel'] = trust_level
+                file['Vendor'] = vendor
+                    # TODO Check if I need two entries for DBotScore
+                    # {'Indicator': file_hash, 'Type': 'file', 'Vendor': tl_score['vendor'],
+                    #     'Score': tl_score['score'], 'Reliability': demisto.params().get('integrationReliability')}
+
+                if score >= 2:
+                    file['Malicious'] = {'Vendor': vendor,
+                                         'Score': score,
+                                         'Description': 'Trust level is ' + str(tl_score['trust_level'])
+                                         }
+                    dbot_score_instance.malicious_description = f"Trust level is {str(tl_score['trust_level'])}"
+            if(hash_type_key == HashType.MD5):
+                file_instance.md5 = file_hash
+            elif(hash_type_key == HashType.SHA1):
+                file_instance.sha1 = file_hash
+            elif(hash_type_key == HashType.SHA256):
+                file_instance.sha256 = file_hash
+
+            entry_context = {'DBotScore': dbot_score, 'File': file}
+            table = reputations_to_table(reputations)
+            command_results.append(
+                CommandResults(readable_output=tableToMarkdown('McAfee TIE Hash Reputations For %s:' % (file_hash,), table),
+                               raw_response=raw_result,
+                               outputs_prefix='McAfee.TIE',
```

Comment:
i'd put the prefix as constant, as it's the same for all `CommandResult` objects we'll create here